.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "experimental/packages/otlp-grpc-exporter-base/protos"]
-	path = experimental/packages/otlp-grpc-exporter-base/protos
-	url = https://github.com/open-telemetry/opentelemetry-proto.git
 [submodule "experimental/packages/otlp-proto-exporter-base/protos"]
```

Comment:
Note: we'll be able to remove this submodule here too in the follow-up to do the same to the browser exporters.